webpack.config.js

```diff
@@ -11,6 +11,12 @@ Encore
     .addEntry('authentication', './src/AppBundle/Resources/javascript/authentication.ts')
     .addEntry('registration', './src/AppBundle/Resources/javascript/registration.ts')
 
+    // Disable minification to support ie8
+    .configureUglifyJsPlugin(function (options) {
+        options.ie8 = true;
+        options.compress = false;
```

Comment:
Reading about the --ie8 argument in the man pages, I learned that the compress option also takes an ie8 setting.

This might be worth taking a look at, as we could still compress the file but stay IE8 compatible?

https://github.com/mishoo/UglifyJS2

Reply:
Disabled the `screw_ie8` options in all UglifyJs options.